gk/co.h

```diff
@@ -0,0 +1,182 @@
+/*
+ * Gatekeeper - DoS protection system.
+ * Copyright (C) 2016 Digirati LTDA.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _GATEKEEPER_GK_CO_H_
+#define _GATEKEEPER_GK_CO_H_
+
+#include <stdbool.h>
+#include <string.h>
+#include <aco.h>
+#include <list.h>
+
+#include "gatekeeper_gk.h"
+#include "gatekeeper_acl.h"
+
+/* Information transfered between tasks assigned to the same coroutine. */
+struct gk_co_leftover {
+	struct flow_entry *fe;
+	struct gk_fib     *fib;
```

Comment:
This field is not used anywhere.

Reply:
It'll be used when the processing of the back packets are moved to coroutines.